src/test/java/edu/cornell/kfs/fp/batch/xml/fixture/AccountingXmlDocumentBackupLinkFixture.java

```diff
@@ -3,21 +3,33 @@
 import edu.cornell.kfs.fp.batch.xml.AccountingXmlDocumentBackupLink;
 
 public enum AccountingXmlDocumentBackupLinkFixture {
-    CORNELL_INDEX_PAGE("http://www.cornell.edu/index.html", "Cornell index page"),
-    DFA_INDEX_PAGE("https://www.dfa.cornell.edu/index.cfm", "DFA index page");
+    CORNELL_INDEX_PAGE("http://www.cornell.edu/index.html", "Cornell index page", "index.html"),
+    DFA_INDEX_PAGE("https://www.dfa.cornell.edu/index.cfm", "DFA index page", "index.cfm"),
+    AWS_BILLING_INVOICE("https://kfsaws-support.cd.cucloud.net/v1_0/aws/billing/invoice?year=2017&month=01&account=999999999999", "AWS Billing Invoice", 
+            "invoice.pdf", "AWS-Bill");
 
-    public final String linkUrl;
-    public final String description;
+    public String linkUrl;
+    public String description;
+    public String fileName;
+    public String groupCode;

```

Comment:
To avoid the risk of accidentally editing the enum fields at runtime, I would suggest declaring these fields as final. Then you can adjust the four-arg constructor to initialize all four fields, and have the three-arg constructor invoke the four-arg one using an appropriate default or null/empty value for the fourth arg.

Reply:
ok